### DISCUSSION TITLE
Applicaton switcher (Alt-Tab) not working on LXQT 2.0 with KWIN

### DISCUSSION THREAD

**Discussion by Nudnikde** [lxqt/lxqt#2584] [Q&A] [answered]
Dear LXQT Community,

I searched the forum and the net but I couldn’t find anything. I’m on Manjaro 24.0.2 stable branch. On LXQT 2.0 with KWin_x11 I’m encoutering a strange behaviour. The application switcher is not working when I press ALT-TAB. When I open KDE system settings I see the Short Cuts are not activated. When I activate Alt-Tab it is still not working. When I re-enter KDE System Settings I find the keyboard shortcut empty again. It seems the settings are not saved.

Btw. when I log in to a KDE Plasma wayland session everything is working fine.

Maybe s.o. can point me to the right direction?

Thanks

**Comment by Chiitoo** [comment, marked as answer]
Probably don't have `kglobalacceld` auto-starting any longer.

Try running it (as the user) `/usr/libexec/kglobalacceld &` for example, or where ever it is installed for you.

Can set it to auto-start then from session settings.

I haven't tracked down why it is no longer being run, possibly due to missing some KDE component (I think it started for me with KF5 already, where I needed to visit the KDE system settings to get alt-tab working, however I did not know about `kglobalacceld` back then yet and I'm not sure if it was actually the case).

**Reply by stefonarch** [reply]
Exactly. See  https://github.com/lxqt/lxqt/wiki/ConfigWindowManagers#shortcuts-1
It's started automatically by plasma x11 but only there afaik, shortcuts on wayland are directly controlled by kwin_wayland.

**Reply by Nudnikde** [reply]
Great, that solved it.  kglobalacceld hasn't been autostarted!

Thanks a lot!
